mne/io/kit/tests/test_kit.py

```diff
@@ -36,6 +38,8 @@
 hsp_txt_path = op.join(data_dir, 'test_hsp.txt')
 elp_path = op.join(data_dir, 'test.elp')
 hsp_path = op.join(data_dir, 'test.hsp')
+# this is a fake headshape made up of a unit sphere to test decimation
+sphere_hsp_txt_path = op.join(data_dir, 'test_sphere_hsp.txt')
```

Comment:
This is no longer used so should be deleted. But we can fix this in a follow-up PR